supplier/tests/test_supplier.py

```diff
@@ -1,64 +1,23 @@
 import pytest
-from conftest import Fields, TestFields
-from django.core.exceptions import FieldDoesNotExist, ValidationError, ObjectDoesNotExist
+from django.core.exceptions import ObjectDoesNotExist
 from supplier.models import Supplier
 
 
 class TestSupplierModel:
     @pytest.mark.django_db()
-    def test_save_supplier(self, supplier0):
-        assert supplier0 not in Supplier.objects.all()
-        Supplier.save_supplier(supplier0)
-        assert supplier0 in Supplier.objects.all()
+    def test_save_supplier(self, saved_supplier0):
+        assert saved_supplier0 in Supplier.objects.all()
 
     @pytest.mark.django_db()
-    def test_delete_not_existed_supplier(self, supplier0):
+    def test_delete_not_existed_supplier(self, saved_supplier0):
         with pytest.raises(ObjectDoesNotExist):
-            assert supplier0 not in Supplier.objects.all()
-            Supplier.delete_supplier(supplier0)
-
-    @pytest.mark.django_db()
-    def test_delete_exist_supplier(self, supplier0):
-        assert supplier0 not in Supplier.objects.all()
-        Supplier.save_supplier(supplier0)
-        assert supplier0 in Supplier.objects.all()
-        Supplier.delete_supplier(supplier0)
-        assert supplier0 not in Supplier.objects.all()
-
-    @pytest.mark.django_db()
-    def test_fail_to_save_supplier(self, supplier0):
-        with pytest.raises(ValidationError):
-            supplier0.user_name = ''
-            Supplier.save_supplier(supplier0)
```

Comment:
we need this test, business field has validators, you can return the test and change the test field to business name

Reply:
Fixed!